_modules/Week 5 -  Digging into a digital common : the case of Decidim

```diff
@@ -0,0 +1,10 @@
+title: Week 5 -  Digging into a digital common : the case of Decidim
+---
+
+Slides
+: **PDF**{:  }
+  : [Download](/assets/pdfs/Digital%20Commons%20and%20Public%20Policy%20-%20Week%201.pdf)
```

Comment:
This the first course slides. Since for Week5 there's no bibliography or slides (yet). You can comment the whole section (which will have it not display) by adding `<!--` at the beginning of line 6 and `-->` at the end of line 10.